styles/designs/carnival/parts/_toc-components.scss

```diff
@@ -139,7 +139,6 @@ $Toc__List--TopLevel: (
     list-style: none,
     padding-left: 0,
     margin-left: 0,
-    display: inline,
```

Comment:
Very odd case :thinking: I'm not sure why this property was added to the main ol from TOC, maybe it's a leftover from old styles. It doesn't change anything, but for some reason because of that property I coudn't remove margins from its list items :eyes: What is more weird, only in PDFs, in baked HTML my changes were visible.